src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -15,64 +16,45 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
     /// <summary>
     /// A TupleTypeSymbol represents a tuple type, such as (int, byte) or (int a, long b).
     /// </summary>
-    internal sealed class TupleTypeSymbol : NamedTypeSymbol
+    internal sealed class TupleTypeSymbol : WrapperNamedTypeSymbol
     {
-        private readonly NamedTypeSymbol _underlyingType;
-        private readonly ImmutableArray<TupleFieldSymbol> _fields;
-        private readonly bool _hasFriendlyNames;
+        private readonly ImmutableArray<Location> _locations;
+        private readonly ImmutableArray<Location> _elementLocations;
```

Comment:
_elementLocations are declaration locations for elements. Similar to declaration locations for fields of Anonymous Types. I'll add a comment in the next change.

---
In reply to: [61785892](https://github.com/dotnet/roslyn/pull/11006#discussion_r61785892) [](ancestors = 61785892)